code/modules/mob/living/carbon/alien/larva/larva.dm

```diff
@@ -26,38 +26,6 @@
 
 	..()
 
-//This is fine, works the same as a human
-/mob/living/carbon/alien/larva/Bump(atom/movable/AM as mob|obj, yes)
-
-	spawn( 0 )

```

Comment:
No wonder the game starts lagging when there's a bunch of larva in viro all bopping all over.